CHANGELOG.md

```diff
@@ -57,4 +57,5 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 - [#70](https://github.com/thanos-io/objstore/pull/70) GCS: Update cloud.google.com/go/storage version to `v1.27.0`.
 - [#71](https://github.com/thanos-io/objstore/pull/71) Replace method `IsCustomerManagedKeyError` for a more generic `IsAccessDeniedErr` on the bucket interface.
 - [#89](https://github.com/thanos-io/objstore/pull/89) GCS: Upgrade cloud.google.com/go/storage version to `v1.35.1`.
+- [#123](https://github.com/thanos-io/objstore/pull/123) *: Upgrade minio-go version to `v7.0.71`.
```

Comment:
Upgraded version was wrong in the changelog

Reply:
My bad: https://github.com/thanos-io/objstore/pull/124 PR to fix it